eng/Publishing.props

```diff
@@ -46,12 +48,12 @@
   </ItemGroup>
 
```

Comment:
Why havent we migrated Build.props yet? Is that needed from the installer repo? I'm not sure where the installer build is added within the pipeline, I was unable to find it.